Trimming leading and tailing white spaces in crt file to avoid SSL issue

Praveenrajmani opened the pull request:
Issue Fix: SSL - **Invalid SSL certificate file #5632**

The spaces in the public.crt throws "Invalid SSL certificate file"

Changes Done: Removed the tailing/heading spaces in **parsePublicCertFile** function #

aead reviewed (approved):
LGTM - but code must be formatted properly with `gofmt`

Praveenrajmani committed:
Removed the tailing/heading spaces in parsePublicCertFile function

The spaces in the public.crt throws "Invalid SSL certificate file"

Resolves: Invalid SSL certificate file #5632
